callinmullaney/decanter
# feat: Add font-display swap support to font loading

# READY FOR REVIEW

# Summary
- Adds `font-display="swap"` css property to font loading in order to increase site performance and lighthouse scores

# Needed By (Date)
- ASAP

# Urgency
- low

# Steps to Test

1. Review each google font link imported in CSS. Verify the corresponding stylesheets load with the `@font-face` selector including the `font-display: swap` property.

# Affected Projects or Products
- Stanford HumSci platform. Various performance audits recommend this `font-display: swap` value added to `@font-face` in order to increase font loading - see https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display and https://www.smashingmagazine.com/2021/05/reduce-font-loading-impact-css-descriptors/

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
